### PR TITLE
change field name from feedInstanceReliability to feedReliability

### PR DESCRIPTION
**Pull request by yuvalbenshalom**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/21724

## Description
change field name from feedInstanceReliability to feedReliability to comply with server standard.
we will need to add validation for this. 

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [x] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No - because 5.5.0 was not released.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/FeedCSV/Integrations/FeedCSV/FeedCSV.yml [hunk 1]
```diff
@@ -28,39 +32,46 @@ configuration:
...
 - display: Username
   name: credentials
   required: false
   type: 9
-- display: Trust any certificate (not secure)
+- defaultvalue: ''
+  display: Trust any certificate (not secure)
   name: insecure
   required: false
   type: 8
```
Comment (avidan-H) [inline Packs/FeedCSV/Integrations/FeedCSV/FeedCSV.yml]: Let's put `proxy` and `insecure` at the bottom of the YAML parameters.

File: Integrations/FeedTAXII/FeedTAXII.yml [hunk 2]
```diff
@@ -25,24 +33,31 @@ configuration:
...
+    this feed. This means that if an indicator from this feed is on the exclusion
+    list, the indicator might still be added to the system.
+  display: Bypass exclusion list
+  name: feedBypassExclusionList
   required: false
-  additionalinfo: When selected, the exclusion list will be ignored for this feed.
+  type: 8
 - defaultvalue: ''
   display: Trust any certificate (not secure)
   name: insecure
```
Comment (avidan-H) [inline Integrations/FeedTAXII/FeedTAXII.yml]: Same here. Let's put `insecure` and `proxy` at the bottom.

File: Integrations/FeedOffice365/FeedOffice365.yml [hunk 3]
```diff
@@ -64,39 +54,40 @@ configuration:
   - F - Reliability cannot be judged
   required: true
   type: 15
-- display: ""
-  name: expiration
-  defaultvalue: ""
+- defaultvalue: indicatorType
```
Comment (avidan-H) [inline Integrations/FeedOffice365/FeedOffice365.yml]:
```suggestion
- defaultvalue: indicatorType
  display: ""
```

File: Packs/FeedSpamhaus/Integrations/FeedSpamhaus/FeedSpamhaus.yml [hunk 4]
```diff
@@ -44,25 +54,16 @@ configuration:
   type: 8
 - additionalinfo: Indicators from this integration instance will be marked with this
     reputation.
-  defaultvalue: Bad
+  defaultvalue: feedInstanceReputationNotSet
   display: Indicator reputation
-  name: feedReputation
+  name: feedInstanceReputation
```
Comment (avidan-H) [inline Packs/FeedSpamhaus/Integrations/FeedSpamhaus/FeedSpamhaus.yml]:
```suggestion
  name: feedReputation
```

File: Packs/FeedSpamhaus/Integrations/FeedSpamhaus/FeedSpamhaus.yml [hunk 5]
```diff
@@ -44,25 +54,16 @@ configuration:
   type: 8
 - additionalinfo: Indicators from this integration instance will be marked with this
     reputation.
-  defaultvalue: Bad
+  defaultvalue: feedInstanceReputationNotSet
   display: Indicator reputation
```
Comment (avidan-H) [inline Packs/FeedSpamhaus/Integrations/FeedSpamhaus/FeedSpamhaus.yml]:
```suggestion
  display: Indicator Reputation
```

File: Packs/FeedSpamhaus/Integrations/FeedSpamhaus/FeedSpamhaus.yml [hunk 6]
```diff
@@ -44,25 +54,16 @@ configuration:
   type: 8
 - additionalinfo: Indicators from this integration instance will be marked with this
     reputation.
```
Comment (avidan-H) [inline Packs/FeedSpamhaus/Integrations/FeedSpamhaus/FeedSpamhaus.yml]:
```suggestion
    reputation
```

File: Packs/FeedSpamhaus/Integrations/FeedSpamhaus/FeedSpamhaus.yml [hunk 7]
```diff
@@ -44,25 +54,16 @@ configuration:
...
-  hidden: true
-  name: feedExpirationPolicy
-  required: false
-  type: 0
-- hidden: true
-  name: feedExpirationInterval
-  required: false
-  type: 1
 - display: Trust any certificate (not secure)
   name: insecure
```
Comment (avidan-H) [inline Packs/FeedSpamhaus/Integrations/FeedSpamhaus/FeedSpamhaus.yml]: `insecure` and `proxy` params to the bottom please.

File: Packs/FeedSpamhaus/Integrations/FeedSpamhaus/FeedSpamhaus.yml [hunk 8]
```diff
@@ -32,9 +32,19 @@ configuration:
   - F - Reliability cannot be judged
   required: false
   type: 15
-- name: expiration
+- defaultvalue: indicatorType
```
Comment (avidan-H) [inline Packs/FeedSpamhaus/Integrations/FeedSpamhaus/FeedSpamhaus.yml]:
```suggestion
- display: ""
  defaultvalue: indicatorType
```

File: Packs/FeedSpamhaus/Integrations/FeedSpamhaus/FeedSpamhaus.yml [hunk 9]
```diff
@@ -32,9 +32,19 @@ configuration:
   - F - Reliability cannot be judged
   required: false
```
Comment (avidan-H) [inline Packs/FeedSpamhaus/Integrations/FeedSpamhaus/FeedSpamhaus.yml]:
```suggestion
  required: true
```

File: Packs/FeedSpamhaus/Integrations/FeedSpamhaus/FeedSpamhaus.yml [hunk 10]
```diff
@@ -44,25 +54,16 @@ configuration:
   type: 8
```
Comment (avidan-H) [inline Packs/FeedSpamhaus/Integrations/FeedSpamhaus/FeedSpamhaus.yml]:
```suggestion
  defaultvalue: ""
  type: 8
```

**Comment by avidan-H** [comment]
Please, also delete the context outputs for `csv-get-indicators` command. It should also be removed from the code if it's there but if you don't want to include that change in this PR, that's fine.

**Comment by liorblob** [comment]
@avidan-H Removing context outputs might break the test playbook, so it would need to be fixed as well.

File: Packs/FeedOffice365/Integrations/FeedOffice365/FeedOffice365.yml [hunk 11]
```diff
@@ -30,60 +29,59 @@ configuration:
...
   - Good
   - Suspicious
   - Bad
-  additionalinfo: Indicators from this integration instance will be marked with this
-    reputation
-- display: Source Reliability
+  required: false
+  type: 18
+- additionalinfo: Reliability of the source providing the intelligence data
+  defaultvalue: F - Reliability cannot be judged
```
Comment (ronykoz) [inline Packs/FeedOffice365/Integrations/FeedOffice365/FeedOffice365.yml]: should be A

File: Integrations/FeedTAXII/FeedTAXII.yml [hunk 12]
```diff
@@ -8,14 +8,22 @@ configuration:
   name: feed
```
Comment (avidan-H) [inline Integrations/FeedTAXII/FeedTAXII.yml]:
```suggestion
  name: feed
  defaultvalue: true
```

File: Packs/FeedCSV/Integrations/FeedCSV/FeedCSV.yml [hunk 13]
```diff
@@ -3,22 +3,26 @@ commonfields:
   id: CSVFeed
   version: -1
 configuration:
-- display: URL
-  name: url
-  required: true
-  type: 0
 - display: Fetch indicators
   name: feed
```
Comment (avidan-H) [inline Packs/FeedCSV/Integrations/FeedCSV/FeedCSV.yml]:
```suggestion
  name: feed
  defaultvalue: true
```

File: Packs/FeedOffice365/Integrations/FeedOffice365/FeedOffice365.yml [hunk 14]
```diff
@@ -5,9 +5,8 @@ commonfields:
 configuration:
 - display: Fetch indicators
   name: feed
```
Comment (avidan-H) [inline Packs/FeedOffice365/Integrations/FeedOffice365/FeedOffice365.yml]:
```suggestion
  name: feed
  defaultvalue: true
```

File: Packs/FeedOffice365/Integrations/FeedOffice365/FeedOffice365.yml [hunk 15]
```diff
@@ -30,60 +29,59 @@ configuration:
...
+  required: false
+  type: 17
+- defaultvalue: '20160'
   name: feedExpirationInterval
-  defaultvalue: "20160"
-  type: 1
   required: false
-- display: Feed Fetch Interval
+  type: 1
+- defaultvalue: '240'
```
Comment (avidan-H) [inline Packs/FeedOffice365/Integrations/FeedOffice365/FeedOffice365.yml]:
```suggestion
- defaultvalue: '30'
```

File: Packs/FeedOffice365/Integrations/FeedOffice365/FeedOffice365.yml (hunk 11 above)
Comment (DeanArbel) [inline Packs/FeedOffice365/Integrations/FeedOffice365/FeedOffice365.yml]:
```suggestion
  defaultvalue: A - Completely reliable
```

File: Integrations/FeedTAXII/FeedTAXII.yml [hunk 16]
```diff
@@ -25,31 +34,29 @@ configuration:
...
+  - interval
+  - indicatorType
+  - suddenDeath
   required: false
-  type: 0
-- hidden: true
-  name: expirationInterval
+  type: 17
+- defaultvalue: '20160'
+  name: feedExpirationInterval
```
Comment (avidan-H) [inline Integrations/FeedTAXII/FeedTAXII.yml]:
```suggestion
  name: feedExpirationInterval
  display: ""
```

File: Packs/FeedCSV/Integrations/FeedCSV/FeedCSV.yml [hunk 17]
```diff
@@ -28,46 +33,44 @@ configuration:
...
+  options:
+  - never
+  - interval
+  - indicatorType
+  - suddenDeath
   required: false
   type: 17
-- display: expirationInterval
+- defaultvalue: '20160'
   name: feedExpirationInterval
```
Comment (avidan-H) [inline Packs/FeedCSV/Integrations/FeedCSV/FeedCSV.yml]:
```suggestion
  name: feedExpirationInterval
  display: ""
```

**Comment by DeanArbel** [comment]
> Removing the context outputs of `FeedCSV` `csv-get-indicators` command - are you leaving that for a separate PR because it requires updating the code of the CSV module and test playbook?

yes

**Pull request merged**